svgReact/icons/getColorContrast.ts

```diff
@@ -0,0 +1,35 @@
+const getRgbFromColor = (color: string) => {
+  // Check the format of the color, HEX or RGB?
+  if (color.match(/^rgb/)) {
+    // If RGB --> store the red, green, blue values in separate variables
+    const [, r, g, b] =
+      color.match(
+        /^rgba?\((\d+),\s*(\d+),\s*(\d+)(?:,\s*(\d+(?:\.\d+)?))?\)$/
+      ) || [];
+    return { r: Number(r), g: Number(g), b: Number(b) };
+  }
+  // If hex --> Convert it to RGB: http://gist.github.com/983661
+  const rgbColor = +`0x${color
+    .slice(1)
+    .replace(color.length < 5 ? /./g : '', '$&$&')}`;
+  return {
+    // eslint-disable-next-line no-bitwise
+    r: rgbColor >> 16,
+    // eslint-disable-next-line no-bitwise
+    g: (rgbColor >> 8) & 255,
+    // eslint-disable-next-line no-bitwise
+    b: rgbColor & 255,
+  };
+};
+
+export const getHspFromColor = (
```

Comment:
Have a functon to get constrast of the color to know which color is lighter, which color is darker are really cool. Uses and apply this function to icon generation are really good idea! 💯